resources/user-agents/apps/facebook/facebook-app.json

```diff
@@ -651,17 +662,6 @@
         {
           "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) Version/* Safari/*?FB*/*;FBAV/*",
           "devices": {
-            "GT-I8190": "Samsung GT-I8190"
-          },
-          "engine": "WebKit",
-          "platforms": [
-            "Android_4_3", "Android_4_2", "Android_4_1", "Android_4_0",
```

Comment:
When looking at coverage I noticed this one didn't have these platforms covered, so looked up the device. Apparently this device only supported Android 4.1.

https://en.wikipedia.org/wiki/Samsung_Galaxy_S_III_Mini